10 Ditto/10Ditto.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+  </head>
+  <body>
+
+    <script>
+    let obj1 = {thing : "hi",
+            Game : "Neighbor",
+            pokemon : "peecachu"}
+
+    let obj2 = {thing : "Neighbor",
```

Comment:
Proper convention is that the key and the colon sign should be next to each other.

```js
const obj2 = { thing: "Neighbor", Game: "Neighbor" };
```